src/api/firebase.js

```diff
@@ -140,23 +140,31 @@ export async function createList(userId, userEmail, listName) {
  * @param {string} recipientEmail The email of the user to share the list with.
  */
 export async function shareList(listPath, currentUserId, recipientEmail) {
-	// Check if current user is owner.
-	if (!listPath.includes(currentUserId)) {
-		return;
-	}
-	// Get the document for the recipient user.
-	const usersCollectionRef = collection(db, 'users');
-	const recipientDoc = await getDoc(doc(usersCollectionRef, recipientEmail));
-	// If the recipient user doesn't exist, we can't share the list.
-	if (!recipientDoc.exists()) {
-		return;
+	try {
+		// Check if current user is owner.
+		if (!listPath.includes(currentUserId)) {
+			throw new Error(
+				'Current user is not the owner of the list. If this is your list, make sure you are signed in.',
+			);
```

Comment:
This looks good for now. I think it might be a good idea to create a ticket, in the final two weeks of the project, where the user only sees the `Home` until they `authenticate` and select a `list`.  That would make this bit of code null and void, but we can discuss this down the line.  For now, this is perfect.

Reply:
Thats a great idea!